crates/graphql/src/schema/objects/wallet.rs

```diff
@@ -29,4 +99,13 @@ impl Wallet {
             .collect::<Result<_, _>>()
             .map_err(Into::into)
     }
+
+    #[graphql(arguments(creators(description = "a list of auction house public keys")))]
+    pub fn nft_counts(
+        &self,
+        _ctx: &AppContext,
+        creators: Option<Vec<PublicKey<NftCreator>>>,
+    ) -> WalletNftCount {
+        WalletNftCount::new(self.address.clone(), creators)
```

Comment:
@ray-kast is the clone needed everywhere? Compiler error is doesn't implement copy.